### PR TITLE
6pm 3 tutor notes page create only

### PR DESCRIPTION
**Pull request by Zwang1680**
This is a subset of our previous task to create an entire front end of the tutor notes page instead only creating the front end for the creation of the tutor notes

### CODE REVIEW

**Review by m-cabello — COMMENTED** [review]
LGTM! Passes all Tests too!

**Review by m-cabello — APPROVED** [review]
LGTM!

**Review by m-cabello — COMMENTED** [review]
LGTM! Good work removing the comments!

**Review by m-cabello — APPROVED** [review]
LGTM! Good work removing the comments!

File: javascript/src/main/App.js [hunk 1]
```diff
@@ -74,6 +76,8 @@ function App() {
           <AuthorizedRoute path="/courses/edit/:courseId" exact component={EditCourse} authorizedRoles={["admin"]} />
           <AuthorizedRoute path="/officeHours/edit/:officeHourId" exact component={EditOfficeHours} authorizedRoles={["admin"]} />
           <AuthorizedRoute path="/tutorAssignments" exact component={TutorAssignments} authorizedRoles={["admin", "member"]} />
+          <AuthorizedRoute path="/tutorNotes" exact component={TutorNotes} authorizedRoles={["admin", "member"]} />
```
[the remark this answers is not in the document]
Comment (pconrad) [reply javascript/src/main/App.js:79]: Done!

**Review by m-cabello — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**